src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -298,19 +319,46 @@ class DemuxFastqs
  @arg(doc="The sequencing center from which the data originated") val sequencingCenter: Option[String] = None,
  @arg(doc="Predicted median insert size, to insert into the read group header") val predictedInsertSize: Option[Integer] = None,
  @arg(doc="Platform model to insert into the group header (ex. miseq, hiseq2500, hiseqX)") val platformModel: Option[String] = None,
+ @arg(doc="Platform to insert into the group header (e.g Illumina)") val platform: String = "Illumina",
  @arg(doc="Comment(s) to include in the merged output file's header.", minElements = 0) val comments: List[String] = Nil,
  @arg(doc="Date the run was produced, to insert into the read group header") val runDate: Option[Iso8601Date] = None,
  @arg(doc="The type of outputs to produce.") val outputType: Option[OutputType] = None,
- @arg(doc="Output FASTQs according to Illumina naming standards, for example, for upload to the BaseSpace Sequence Hub") val illuminaStandards: Boolean = false,
  @arg(
    doc=
      """Output all bases (i.e. all sample barcode, molecular barcode, skipped,
         and template bases) for every read with template bases (ex. read one
         and read two) as defined by the corresponding read structure(s).
      """)
- val includeAllBasesInFastqs: Boolean = false
+ val includeAllBasesInFastqs: Boolean = false,
+ @deprecated(message="Use outputStandards instead", since="1.3.0")
+ @arg(doc="Output FASTQs according to Illumina BaseSpace Sequence Hub naming standards.  This is differfent than Illumina naming standards.",
+   mutex=Array("fastqSkipReadNumbers", "fastqIncludeSampleBarcodes", "illuminaFileNames"))
+ val illuminaStandards: Boolean = false,
+ @arg(doc="Include a trailing /1 or /2 for R1 and R2 in the FASTQ read name.", mutex=Array("illuminaStandards"))
+ var fastqSkipReadNumbers: Boolean = false,
+ @arg(doc="Update the sample barcode in the FASTQ header.", mutex=Array("illuminaStandards"))
+ var fastqIncludeSampleBarcodes: Boolean = false,
+ @arg(doc="Name the output files according to the Illumina file name standards.", mutex=Array("illuminaStandards"))
+ var illuminaFileNames: Boolean = false
```

Comment:
Can be `val`s now that we have a `FastqStandards` container class
```suggestion
 val fastqSkipReadNumbers: Boolean = false,
 @arg(doc="Update the sample barcode in the FASTQ header.", mutex=Array("illuminaStandards"))
 val fastqIncludeSampleBarcodes: Boolean = false,
 @arg(doc="Name the output files according to the Illumina file name standards.", mutex=Array("illuminaStandards"))
 val illuminaFileNames: Boolean = false
```